apps/api/src/modules/notifications/dtos/providers/smsTwilio-data.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { IsNotEmpty } from 'class-validator';
+
+export class SmsTwilioDataDto {
+  @IsNotEmpty()
+  to: string;
+
+  @IsNotEmpty()
+  message: string;
```

Comment:
We can probably keep a max length limit of 1600 for message.
https://www.twilio.com/docs/glossary/what-sms-character-limit

Spend 15 minutes and check any other min/max values can be set for `to` field too.